rpc/src/client.rs

```diff
@@ -221,6 +224,31 @@ pub trait Client {
             .await
     }
 
+    /// Poll the `/health` endpoint until it returns a successful result or
+    /// the given `timeout` has elapsed.
+    async fn wait_until_healthy<T>(&self, timeout: T) -> Result<()>
+    where
+        T: Into<Duration> + Send,
+    {
+        let timeout = timeout.into();
+        let poll_interval = Duration::from_millis(200);
+        let mut attempts_remaining = timeout.as_millis() / poll_interval.as_millis();
+
+        while let Err(_) = self.health().await {
```

Comment:
```suggestion
        while self.health().await.is_err() {
```

Quick fix for the clippy suggestion.